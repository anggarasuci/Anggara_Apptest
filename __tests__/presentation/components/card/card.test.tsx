import React from 'react';
import {render} from '@testing-library/react-native';
import CardView from '../../../../src/presentation/components/card/card';

test('renders CardView correctly', () => {
  const {toJSON} = render(<CardView>{}</CardView>);

  expect(toJSON()).toMatchSnapshot();
});
