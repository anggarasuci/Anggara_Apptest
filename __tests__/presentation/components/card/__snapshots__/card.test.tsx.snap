// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders CardView correctly 1`] = `
<View
  accessibilityState={
    {
      "busy": undefined,
      "checked": undefined,
      "disabled": undefined,
      "expanded": undefined,
      "selected": undefined,
    }
  }
  accessibilityValue={
    {
      "max": undefined,
      "min": undefined,
      "now": undefined,
      "text": undefined,
    }
  }
  accessible={true}
  collapsable={false}
  focusable={true}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    {
      "backgroundColor": "white",
      "borderRadius": 10,
      "elevation": 5,
      "marginHorizontal": 16,
      "marginVertical": 8,
      "opacity": 1,
      "padding": 16,
      "shadowColor": "#000",
      "shadowOffset": {
        "height": 2,
        "width": 0,
      },
      "shadowOpacity": 0.25,
      "shadowRadius": 3.84,
    }
  }
/>
`;
